src/Modules/OrchardCore.Commerce.Payment.Stripe/Abstractions/IStripePaymentIntentService.cs

```diff
@@ -31,7 +31,7 @@ Task<PaymentIntent> GetOrUpdatePaymentIntentAsync(
     /// the payment intent. Check the implementation for more details.
     /// </summary>
     /// <returns>Created Stripe <see cref="PaymentIntent"/>.</returns>
-    Task<PaymentIntent> CreatePaymentIntentAsync(Amount total);
+    Task<PaymentIntent> CreatePaymentIntentAsync(Amount total, string key = null);
```

Comment:
Why is the parameter called `key`? Isn't this always `shoppingCartId`? Please either rename it to `shoppingCartId` or include a `/// <param name="key">` above, and explain why it's different.